packages/lisk-transactions/src/3_vote_transaction.ts

```diff
@@ -300,20 +300,23 @@ export class VoteTransaction extends BaseTransaction {
 			.filter(vote => vote.charAt(0) === PREFIX_UNVOTE)
 			.map(vote => vote.substring(1));
 		const originalVotes = sender.votes || [];
```

Comment:
Here it should be: `sender.votedDelegatesPublicKeys`

Reply:
And i think this [line](https://github.com/LiskHQ/lisk-elements/pull/1129/files#diff-f9ecf85bc7239f90e9bef6de8faeb1deL280) is redundant:

Reply:
Actually I might need to check all the usages of `sender.votes` and change to `votedDelegatesPublicKeys`

Reply:
Had to also update `undoAsset` good catch!